### ISSUE TITLE
Public repos should be disable-able...

### ISSUE THREAD

**Issue by diranged** [SUSE/Portus#765]
As an administrator I should be able to prevent everyone from marking a repo as `public` -- because after #468, a public repo is now fully accessible to anyone on the internet (if our service is out there).

**Comment by mssola** [comment]
Closing since this appears to be a duplicate of #676. Feel free to re-open :wink:

**Comment by mssola** [comment]
@diranged specifically, you can read [this](https://github.com/SUSE/Portus/issues/676#issuecomment-181379589) summing it up.

**Comment by ghostsquad** [comment]
@mssola this should be re-opened. I'd like to be able to prevent only hard-coded (in the config) repos from being public. Having some option to prevent repos from being made public I think was the point of this issue.
